[Quant] Rename node check function

terrychenism opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* __->__ #71805

Summary:

Rename node check fuction

Test Plan:
python test/test_quantization.py TestQuantizeFx
python test/test_quantization.py TestQuantizeFxOps

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D33783235](https://our.internmc.facebook.com/intern/diff/D33783235)

terrychenism committed:
[Quant] Rename node check function
Summary:

Rename node check fuction

Test Plan:
python test/test_quantization.py TestQuantizeFx
python test/test_quantization.py TestQuantizeFxOps

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

terrychenism commented:
@terrychenism has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D33783235).

jerryzh168 commented on torch/ao/quantization/fx/node_check_utils.py:
```diff
@@ -0,0 +1,37 @@
+    'relu_',
+}
+
+def chech_qnode_func(node, modules):
```
probably a typo? chech -> check?

jerryzh168 replied:
I think it would be better to rename this to `is_flow_supported_ops` and rename `is_call_function` to `is_supported_call_function` and same for `is_call_method` and `is_call_module` to be clearer

terrychenism committed:
Update on "[Quant] Rename node check function"


Summary:

Rename node check fuction

Test Plan:
python test/test_quantization.py TestQuantizeFx
python test/test_quantization.py TestQuantizeFxOps

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D33783235](https://our.internmc.facebook.com/intern/diff/D33783235)

[ghstack-poisoned]

terrychenism committed:
Update on "[Quant] Rename node check function"


Summary:

Rename node check fuction

Test Plan:
python test/test_quantization.py TestQuantizeFx
python test/test_quantization.py TestQuantizeFxOps

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D33783235](https://our.internmc.facebook.com/intern/diff/D33783235)

[ghstack-poisoned]

terrychenism committed:
Update on "[Quant] Rename node check function"


Summary:

Rename node check fuction

Test Plan:
python test/test_quantization.py TestQuantizeFx
python test/test_quantization.py TestQuantizeFxOps

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D33783235](https://our.internmc.facebook.com/intern/diff/D33783235)

[ghstack-poisoned]

terrychenism committed:
Update on "[Quant] Rename node check function"


Summary:

Rename node check fuction

Test Plan:
python test/test_quantization.py TestQuantizeFx
python test/test_quantization.py TestQuantizeFxOps

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D33783235](https://our.internmc.facebook.com/intern/diff/D33783235)

[ghstack-poisoned]

terrychenism commented:
@terrychenism has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D33783235).

jerryzh168 reviewed (approved):
lg, thanks!

terrychenism committed:
Update on "[Quant] Rename node check function"


Summary:

Rename node check fuction

Test Plan:
python test/test_quantization.py TestQuantizeFx
python test/test_quantization.py TestQuantizeFxOps

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D33783235](https://our.internmc.facebook.com/intern/diff/D33783235)

[ghstack-poisoned]

terrychenism commented:
@terrychenism has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D33783235).

terrychenism committed:
Update on "[Quant] Rename node check function"


Summary:

Rename node check fuction

Test Plan:
python test/test_quantization.py TestQuantizeFx
python test/test_quantization.py TestQuantizeFxOps

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D33783235](https://our.internmc.facebook.com/intern/diff/D33783235)

[ghstack-poisoned]
